modules/storage-persistent-storage-fsGroup-pod.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * storage/understanding-persistent-storage.adoc
+//* microshift_storage/understanding-persistent-storage-microshift.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="using_fsGroup_pod_{context}"]
+= Changing fsGroup at the pod level
+
+You can set the set the `fsGroupChangePolicy` parameter in a new or existing deployment, and then the pods that it manages will have this parameter value. You can similarly do this for a statefulset. You cannot edit an existing pod to set `fsGroupChangePolicy`; however, you can set this parameter when creating a new pod.
+
+This procedure describes how to set the `fsGroupChangePolicy` parameter in an existing deployment.
+
+.Prerequisites
+
+* Access to the {product-title} console.
+
+.Procedure
+
+To set the `fsGroupChangePolicy` parameter in an existing deployment:
+
+. Click *Workloads* > *Deployments*.
+
+. On the *Deployment* page, click the desired deployment.
+
+. On the *Deployment details* page, click the *YAML* tab.
+
+. Edit the deployment's YAML file under `spec.template.spec.securityContext` as per the following example file:
++
+.Example deployment YAML file setting `fsGroupChangePolicy`
+[source,yaml]
+----
+...
+spec:
+replicas: 3
+selector:
+matchLabels:
+app: my-app
+template:
+metadata:
+creationTimestamp: null
+labels:
+app: my-app
+spec:
+containers:
+- name: container
+image: 'image-registry.openshift-image-registry.svc:5000/openshift/httpd:latest'
+ports:
+- containerPort: 8080
+protocol: TCP
+resources: {}
+terminationMessagePath: /dev/termination-log
+terminationMessagePolicy: File
+imagePullPolicy: Always
+restartPolicy: Always
+terminationGracePeriodSeconds: 30
+dnsPolicy: ClusterFirst
+securityContext:
+    fsGroupChangePolicy: OnRootMismatch <1>
+...
+----
+<1> `OnRootMismatch` specifies skipping recursive permission change, thus helping to avoid pod timeout problems.
```

Comment:
Document both options values, Always & OnRootMismatch
https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#configure-volume-permission-and-ownership-change-policy-for-pods